### DISCUSSION TITLE
Hi dear lunakit modder

### DISCUSSION THREAD

**Discussion by AredinGame** [Amethyst-szs/smo-lunakit#8] [General]
I try to download lunakit thru Modessy loader from homebrew Store But my SMO crashes , Long story short, i need to know how to download lunakit the original way PWWWEEEEEESSSS ☺️👉👈

**Comment by Amethyst-szs** [comment]
Don't install through modyssey loader, grab the most recent build from releases and place it on the root of your SD card, both the LunaKit and atmosphere folders

Also remember that you need SMO v1.0.0 installed or use the downgrader tool
